packages/payment-processor/src/payment/conversion-erc20.ts

```diff
@@ -0,0 +1,85 @@
+import { ContractTransaction, Signer } from 'ethers';
+import { Provider } from 'ethers/providers';
+import { BigNumberish } from 'ethers/utils';
+
+import { proxyChainlinkConversionPath } from '@requestnetwork/smart-contracts';
+import { ClientTypes, ExtensionTypes } from '@requestnetwork/types';
+
+import { _getErc20FeeProxyPaymentUrl } from './erc20-fee-proxy';
+import { _getErc20ProxyPaymentUrl } from './erc20-proxy';
+
+import { ITransactionOverrides } from './transaction-overrides';
+import { getProvider, getSigner } from './utils';
+import { checkErc20Allowance, encodeApproveAnyErc20 } from './erc20';
+
+/**
+ * Processes the approval transaction of a given payment ERC20 to be spent by the swap router,
```

Comment:
Please review your comments to match the methods (look for references to router or fee proxy)